keps/sig-apps/3939-allow-replacement-when-fully-terminated/kep.yaml

```diff
@@ -24,7 +24,7 @@ stage: beta
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.29"
+latest-milestone: "v1.31"
```

Comment:
Why this change?

Reply:
Yeah, it wasn't entirely clear to me if I should update it since we make changes which affect it. However, since this is just a bugfix, I think we can stay with the old milestone.